WDL/runtime/download.py

```diff
@@ -202,7 +202,11 @@ def aria2c_downloader(
 def awscli_downloader(
     cfg: config.Loader, logger: logging.Logger, uri: str, **kwargs
 ) -> Generator[Dict[str, Any], Dict[str, Any], None]:
-    inputs: Dict[str, Any] = {"uri": uri, "docker": cfg["download_awscli"]["docker"]}
+    inputs: Dict[str, Any] = {
+        "uri": uri,
+        "docker": cfg["download_awscli"]["docker"],
+        "endpoint": cfg["download_awscli"].get("endpoint"),
```

Comment:
This `get()` throws unless either we provide a non-None default (second argument), or keep endpoint defined (not commented out) in the default.cfg. The default.cfg could just have `endpoint=` to define it to an empty string (but then that has to be detected in the tasks WDL)